cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/executor/ExecutorTest.java

```diff
@@ -307,19 +308,26 @@ private void executeAndVerifyNotification(Collection<ExecutionProposal> proposal
 
   private UserTaskManager getMockUserTaskManager() {
     UserTaskManager mockUserTaskManager = EasyMock.mock(UserTaskManager.class);
-    mockUserTaskManager.markTaskFinishExecution(anyString());
-    EasyMock.expect(mockUserTaskManager.markTaskInExecution(anyObject())).andReturn(null).anyTimes();
+    mockUserTaskManager.taskExecutionFinished(anyString());
```

Comment:
Rather than passing `anyString()` can we actually verify that the `uuid` input is the one used in the test?